core/input/input_event.h

```diff
@@ -172,8 +173,8 @@ class InputEventKey : public InputEventWithModifiers {
 	void set_pressed(bool p_pressed);
 	virtual bool is_pressed() const override;
 
-	void set_keycode(uint32_t p_keycode);
-	uint32_t get_keycode() const;
+	void set_keycode(Key p_keycode);
+	Key get_keycode() const;
 
 	void set_physical_keycode(uint32_t p_keycode);
```

Comment:
Should take `Key` too?

Reply:
Maybe, I was uncertain about this one. I will change it if that's desired.

Reply:
I think it should match `set_keycode` yeah, WDYT @bruvzg?